app/client/src/pages/Applications/CommonElements.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import styled from "styled-components";
+import { Text } from "design-system-old";
+import { TextType } from "design-system-old";
+import { Classes as BlueprintClasses } from "@blueprintjs/core";
+
+type ResourceHeadingProps = React.PropsWithChildren<{
+  isLoading?: boolean;
+}>;
+
+export const CardListContainer = styled.div<{ isMobile?: boolean }>`
+  padding: ${({ isMobile }) => isMobile && `0 16px`};
+  padding-bottom: ${({ isMobile }) => !isMobile && `24px`};
```

Comment:
This can be the below instead:

```suggestion
${({ isMobile }) => isMobile ? `padding: 0 16px` : `padding-bottom: 24px`};
```